modules/examiner/php/editexaminer.class.inc

```diff
@@ -353,22 +369,37 @@ class EditExaminer extends \NDB_Form
      */
     function _validateEditExaminer($values)
     {
+        $DB     = \Database::singleton();
```

Comment:
i'm not too sure about this, but I've been seeing that we prefer calling Factory singleton, $factory->database() instead of Database singleton. Could it also be that the former is better in this case?